packages/blockchain-wallet-v4/src/network/api/debitCard/index.ts

```diff
@@ -0,0 +1,24 @@
+import { ProductType } from 'data/components/debitCard/types'
+
+export default ({ authorizedGet, authorizedPost, nabuUrl }) => {
+  const createDCCard = (productCode: string): string =>
```

Comment:
what does it mean? `Create Debit Card Card`?

Reply:
good catch, removed second card